legal/src/main/resources/legal/DEPENDENCIES

```diff
@@ -66,6 +91,8 @@ maven/mavencentral/io.quarkus/quarkus-development-mode-spi/1.13.7.Final, Apache-
 maven/mavencentral/io.quarkus/quarkus-jackson/1.13.7.Final, Apache-2.0, approved, clearlydefined
 maven/mavencentral/io.quarkus/quarkus-jaeger/1.13.7.Final, Apache-2.0, approved, clearlydefined
 maven/mavencentral/io.quarkus/quarkus-kafka-client/1.13.7.Final, Apache-2.0, approved, clearlydefined
+maven/mavencentral/io.quarkus/quarkus-kubernetes-client/1.13.7.Final, Apache-2.0, approved, clearlydefined
+maven/mavencentral/io.quarkus/quarkus-kubernetes-client-internal/1.13.7.Final, Apache-2.0, approved, clearlydefined
```

Comment:
this looks odd, doesn't it?

Reply:
It looks odd yes. But excluding the dependency will lead to a build error:
`Failed to load steps from class io.quarkus.kubernetes.client.deployment.KubernetesClientBuildStep`